app/products/calls/components/start_call/start_call.tsx

```diff
@@ -29,9 +30,10 @@ type Props = {
 const StartCall = (props: Props) => {
     const {testID, canStartCall, theme, actions, currentChannelId, callChannelName, currentChannelName, confirmToJoin, alreadyInTheCall, ongoingCall, intl} = props;
 
-    const handleStartCall = useCallback(preventDoubleTap(() => {
+    const [tryLeaveAndJoin, msgPostfix] = useTryCallsFunction(() => {
         leaveAndJoinWithAlert(intl, currentChannelId, callChannelName, currentChannelName, confirmToJoin, actions.joinCall);
-    }), [actions.joinCall, currentChannelId]);
+    });
+    const handleStartCall = useCallback(preventDoubleTap(tryLeaveAndJoin), [actions.joinCall, currentChannelId]);
```

Comment:
I guess it also depends on `intl`, `callChannelName` and `currentChannelName` used by `tryLeaveAndJoin`